app/models/immunisation_record.rb

```diff
@@ -11,4 +12,20 @@ class ImmunisationRecord < ApplicationRecord
   def heading
     full_name
   end
+
+  def last_data_update
+    audits.where(comment: update_audit_comment).last
```

Comment:
weirdly not covered in tests -- are we rendering any in the tests?

Reply:
I thought so, but maybe not